web/beacon-app/src/constants/app.ts

```diff
@@ -51,3 +51,7 @@ export enum MIME_TYPE {
   OCTET_STREAM = 'application/octet-stream',
   PARQUET = 'application/parquet',
 }
+
+export const PUBLIC_LINKS = {
```

Comment:
Should this be moved to `EXTERNAL_LINKS` so that it'll be with the other external links?

Reply:
Sure , i forgot about this. thanks